project_internal_access_from_portal/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_project_portal,access.project.portal,project.model_project_project,base.group_portal,1,0,0,0
```

Comment:
Access rights for the portal user are already [there](https://github.com/odoo/odoo/blob/61d3f8460eaf4217d258bab60f1da8d3281accab/addons/project/security/ir.model.access.csv#L17). No need to add them again.